metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -2644,7 +2658,12 @@ def emit_sheets(self) -> Iterable[MetadataWorkUnit]:
             c.SHEETS_CONNECTION,
             sheets_filter,
         ):
-            yield from self.emit_sheets_as_charts(sheet, sheet.get(c.WORKBOOK))
+            if self.config.ingest_hidden_assets or not self._is_hidden_view(sheet):
+                yield from self.emit_sheets_as_charts(sheet, sheet.get(c.WORKBOOK))
+            else:
+                logger.debug(
+                    f"Skip view {sheet.get(c.ID)} because it's hidden (luid is blank)."
+                )
```

Comment:
Can we keep a counter in TableauSourceReport for tracking, say num_hidden_views_skipped to keep track of how many sheets/dashboards were skipped?

Reply:
Good idea, will add that in the upcoming days. Thanks for the review.